llvm/lib/Target/EraVM/EraVMInstrInfo.td

```diff
@@ -96,6 +98,15 @@ def EraVMlog_decommit : SDNode<"EraVMISD::LOG_DECOMMIT", SDT_EraVMLogDecommit, [
 def EraVMTrap : SDNode<"EraVMISD::TRAP", SDTNone,
     [SDNPHasChain, SDNPOptInGlue]>;
 
+// Those instructions will set flag register. The DAG schedule shouldn't schedule another
+// flag-setting instruction betweem them and their users. The original patch uses Glue to
```

Comment:
Please rephrase to not to mention the original patch.

Reply:
Rephrased.